lib/collectors.js

```diff
@@ -44,29 +44,35 @@ module.exports = (api) => {
     }
   };
 
-  api.metasync.DataCollector.prototype.on = function(
-    // DataCollector events:
-    eventName,
-    callback
-    // on('error', function(err, key))
-    // on('timeout', function(err, data))
-    // on('done', function(errs, data))
-    //   errs - hash of errors
-    //   data - hash of sucessfully received adta
-  ) {
-    if (eventName in this.events) {
-      this.events[eventName] = callback;
-    }
-  };
+  const Collector = {
+    on(
+      // Collector events:
+      eventName,
+      callback
+      // on('error', function(err, key))
+      // on('timeout', function(err, data))
+      // on('done', function(errs, data))
+      //   errs - hash of errors
+      //   data - hash of sucessfully received adta
+    ) {
+      if (eventName in this.events) {
+        this.events[eventName] = callback;
+      }
+    },
```

Comment:
Extra commas in such cases are generally useful as they make Git diffs less noisy when something new is added, but I'm fine with or without a comma here and I don't feel strongly for one way or other.